src/components/UserProfile/UserSettings/UserPasswordChange/index.tsx

```diff
@@ -0,0 +1,193 @@
+import axios from 'axios';
+import { Field, Form, Formik } from 'formik';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import { useToasts } from 'react-toast-notifications';
+import useSWR from 'swr';
+import { useUser } from '../../../../hooks/useUser';
+import Error from '../../../../pages/_error';
+import Alert from '../../../Common/Alert';
+import Button from '../../../Common/Button';
+import LoadingSpinner from '../../../Common/LoadingSpinner';
+import * as Yup from 'yup';
+
+const messages = defineMessages({
+  changepassword: 'Change Password',
+  currentpassword: 'Current Password',
+  newpassword: 'New Password',
+  confirmpassword: 'Confirm Password',
+  save: 'Save Changes',
+  saving: 'Saving…',
+  toastSettingsSuccess: 'Password changed!',
+  toastSettingsFailure:
+    'Something went wrong while changing the password. Is your current password correct?',
+  validationCurrentPassword: 'Current password is required.',
+  validationNewPassword: 'A new password is required.',
+  validationNewPasswordLength: 'Password must be at least 8 characters.',
+  validationConfirmPassword: 'You must confirm the password.',
+  validationConfirmPasswordSame: 'Passwords must match.',
```

Comment:
```suggestion
  validationCurrentPassword: 'You must provide the current password',
  validationNewPassword: 'You must provide a new password',
  validationNewPasswordLength: 'Password is too short; should be a minimum of 8 characters',
  validationConfirmPassword: 'You must confirm your new password',
  validationConfirmPasswordSame: 'Password must match',
```

Just to be consistent with strings elsewhere.  We can also update the strings in other places, if you dislike any of these.